perftests/standalone-enc/main.c

```diff
@@ -0,0 +1,450 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "cpusupport.h"
+#include "crypto_aes.h"
+#include "crypto_aesctr.h"
+#include "parsenum.h"
+#include "perftest.h"
+#include "proto_crypt.h"
+#include "sha256.h"
+#include "sysendian.h"
+#include "warnp.h"
+
+/* Smaller buffers are padded, so no point testing smaller values. */
+static const size_t perfsizes[] = {1024};
+static const size_t num_perf = sizeof(perfsizes) / sizeof(perfsizes[0]);
+static const size_t nbytes_perftest = 1000000000;	/* 1000 MB */
+static const size_t nbytes_warmup = 100000000;		/* 100 MB */
```

Comment:
Weirdly inconsistent indentation of the comments here.

Reply:
That's a github display thing.  :(

It's something weird like their "display tab" code counts the "+ " as part of the line, or they just have an off-by-one error somewhere.  The actual c file is fine in this respect.